src/utils/index.js

```diff
@@ -34,3 +34,24 @@ export const escapeCssIdentifier = (ident) => {
 export const toMs = (s) => {
 	return Number(s.slice(0, -1).replace(',', '.')) * 1000;
 };
+
+// Fill versions to exactly 3 decimals
+export const normalizeVersion = (version) => {
+	return String(version)
+		.split('.')
+		.concat([0, 0])
+		.slice(0, 3)
+		.join('.');
+};
+
+// Numerically compare version strings after normalizing them
+export const compareVersion = (a, b) => {
+	a = normalizeVersion(a);
+	b = normalizeVersion(b);
+	return a.localeCompare(b, undefined, { numeric: true });
+};
+
+// Check if the required version or higher is installed
+export const minVersion = (installed, required) => {
+	return compareVersion(installed, required) >= 0;
+};
```

Comment:
Since these are not part of the main bundle (right?), we might as well just use the [semver](https://www.npmjs.com/package/semver) package under the hood? Seem like there should be a overlap in the functionality.

Reply:
The semver package is quite bloated as it has to deal with lots of complex patterns and edge cases. I'm wondering if we should create this at all, actually. Let's discuss tomorrow...